sentinel-adapter/sentinel-apache-dubbo-adapter/src/main/java/com/alibaba/csp/sentinel/adapter/dubbo/DubboUtils.java

```diff
@@ -35,10 +36,10 @@ public static String getApplication(Invocation invocation, String defaultValue)
 
     public static String getResourceName(Invoker<?> invoker, Invocation invocation) {
         StringBuilder buf = new StringBuilder(64);
-        buf.append(invoker.getInterface().getName())
-                .append(":")
-                .append(invocation.getMethodName())
-                .append("(");
+        buf.append(invoker.getUrl().getEncodedServiceKey())
```

Comment:
> Friendly ping :)

最近工作有点忙... 没及时响应，抱歉

Reply:
> 细分粒度吧，`getEncodedServiceKey`细分为interfaceName+group+version

This could bring breaking changes, which is not recommended. Actually we've had a discussion before: #67 

In most scenarios, the groupName and version are not necessary for flow control. Maybe we could add a property to control it.

Reply:
我也赞同#67这个人的回答，为啥不把group和version作为resourceName的一部分呢？当用户决定使用group和version属性的时候，往往这也表示这是对同一个java的接口多种实现，难道不应该区分开来吗

Reply:
We could add a property configuration to control this (e.g. in [DubboConfig](https://github.com/alibaba/Sentinel/blob/master/sentinel-adapter/sentinel-apache-dubbo-adapter/src/main/java/com/alibaba/csp/sentinel/adapter/dubbo/config/DubboConfig.java)), but I don't think it's a good idea to change the default behavior, as this could bring breaking changes, which will break existing rules of users.

Reply:
你是说不兼容的问题吗，确实应该设置一个开关去让用户选择使用`getEncodedServiceKey`或者是`getInterface`